.babelrc

```diff
@@ -8,32 +8,51 @@
         "istanbul"
       ]
     },
+
     "development": {
       "presets": ["airbnb"],
       "plugins": [
         "inline-react-svg",
         ["transform-replace-object-assign", { "moduleSpecifier": "object.assign" }],
       ],
     },
+
     "production": {
       "presets": ["airbnb"],
       "plugins": [
         "inline-react-svg",
         ["transform-replace-object-assign", { "moduleSpecifier": "object.assign" }],
+        ["transform-react-remove-prop-types", {
```

Comment:
is this something we could add in babel-preset-airbnb, possibly behind an option, instead of in individual repos?

Reply:
Sure, I think so!

I think we'd still need to make it configurable though, since in some places we want the mode to be remove instead of wrap for instance, and other options might be important to manage on a case by case basis, but we could just have a passthrough option.

Reply:
sure, a passthrough option sounds great